source/manuals/monitor-paas-app-with-prometheus.html.md.erb

```diff
@@ -33,12 +33,24 @@ You can create a Prometheus service within your PaaS space and allow it to bind
   - `cf create-service gds-prometheus prometheus <service-instance-name>`
 2. Either [update your app's manifest.yml] to bind your new service or bind it using the CLI
   - `cf bind-service <app-name> <service-instance-name>`
+3. If your app uses a custom domain, you need to make sure that the [authorization header and other app headers][] are being forwarded otherwise the basic auth to the metrics endpoint will fail and your app won't have access to your other headers.
+  - `cf update-service <cdn-route-service> \
+    -c '{"domain": "<custom-domain>", "headers": ["Accept", "Authorization", "<other app header values>"]}'`
 
-Within 5 minutes Prometheus will start scraping your application for metrics, you can validate this by checking [Grafana](https://grafana-paas.cloudapps.digital).
+Within 5 minutes Prometheus will start scraping your application for metrics, you can validate this by checking [Grafana][].
 
+### Notes about the app route
```

Comment:
Can we change this from a heading to a lead-in line for the bullets below, I think this would work better. Do the bullets apply to just the custom domain step or all three steps?

Reply:
OK, so drop the heading and just have it as a lead-in line.
The bullets apply to all domains, regardless if they are custom or not.